packages/web/components/Setup/SetupProfilePicture.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Button,
+  Center,
+  Flex,
+  Image as ChakraImage,
+  Input,
+  VStack,
+} from '@metafam/ds';
+import OctoAvatar from 'assets/graphics/octo-avatar.svg';
+import React, { useRef, useState } from 'react';
+
+import { ProfileWizardPane } from './ProfileWizardPane';
+import { WizardPaneCallbackProps } from './WizardPane';
+
+type ImageData = {
+  file: File;
+  width: number;
+  height: number;
+};
+
+export const SetupProfilePicture: React.FC = () => {
+  const field = 'profileImageURL';
+  const [, setImage] = useState<ImageData | null>();
+  const [preview, setPreview] = useState<string | null>();
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  return (
+    <ProfileWizardPane
+      {...{ field }}
+      title="Profile Picture"
+      prompt="Upload an image that will make you instantly recognizable."
+    >
+      {({ register, setter, current }: WizardPaneCallbackProps) => {
+        const { ref: registerRef } = register(field, {});
+
+        const handleUpload = async (e: React.ChangeEvent<HTMLInputElement>) => {
+          const imageFile = e.target.files?.[0];
+          if (!imageFile) return;
+
+          const img = new Image();
+          img.src = URL.createObjectURL(imageFile);
+          img.onload = () => {
+            const imgData = {
+              file: imageFile,
+              width: img.width,
+              height: img.height,
+            };
+            setImage(imgData);
+            setPreview(URL.createObjectURL(imageFile));
+            // need to pass the height and width to the setter in order to pass it to ceramic
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            setter(imgData);
+          };
+        };
+        const handleClick = () => {
+          inputRef.current?.click();
+          inputRef.current?.focus();
+        };
+        const handleRemove = () => {
+          setImage(null);
+          setter('');
+          setPreview(null);
+        };
+
+        return (
+          <VStack mt={5}>
+            <Center mb="6">
+              <ChakraImage
+                height="200px"
+                width="200px"
+                objectFit="cover"
+                borderRadius="full"
+                src={preview || current || OctoAvatar.src}
+              />
+            </Center>
+            <Input
+              type="file"
+              onChange={handleUpload}
+              name="profileImageURL"
+              accept="image/png, image/jpeg, image/gif"
```

Comment:
```suggestion
              accept="image/*"
```

This excludes, SVGs, WEBPs, AVIFs, and other valid image types.

There's a small chance that someone will try and enter BMP or something bizarre, but the chance is small.